### ISSUE TITLE
NTR: alpha-ketoglutarate dehydrogenase activity in fibroblasts low (Lab: blood chemistry: enzymes)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8233]
New term request alpha-ketoglutarate dehydrogenase activity in fibroblasts low (elthat_180227112931): alpha-KGDH  UMLS: 4229108

**Comment by pnrobinson** [comment]
@MickeySegal
does this refer to the alpha-ketoglutarate dehydrogenase complex? (see [OMIM](https://omim.org/entry/613022))
Is this a clinical assay?

**Comment by MickeySegal** [comment]
That is correct, and also in:
https://www.omim.org/entry/617668
"Patient fibroblasts showed decreased lipoic acid-dependent enzymatic activities of the pyruvate dehydrogenase complex (PDHc), alpha-ketoglutarate dehydrogenase complex (KGDHc), and branched-chain ketoacid dehydrogenase complex (BCKDHc)."

https://www.omim.org/entry/615330
"Biochemical evaluation of patient tissues showed mitochondrial defects, particularly of respiratory complexes I, II, and IV and the lipoate-containing enzymes alpha-ketoglutarate dehydrogenase and pyruvate dehydrogenase."

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000830
- label: Diminished alpha-ketoglutarate dehydrogenase activity in cultured fibroblasts
- parents: [Abnormal enzyme activity in cultured fibroblasts (HP:4000195)](https://hpo.jax.org/app/browse/term/HP:4000195)
- pmid: 
- def: Concentration or activity of the alpha-ketoglutarate dehydrogenase complex as measured in cultured fibroblasts is below the limits of normal. This enzyme is a multienzyme complex consisting of 3 protein subunits, oxoglutarate dehydrogenase, also known as alpha-ketoglutarate dehydrogenase or E1k (EC 1.2.4.2.), dihydrolipoyl succinyltransferase (DLST, or E2k; 126063), and dihydrolipoyl dehydrogenase (DLD, or E3; 238331). The complex catalyzes a key reaction in the Krebs tricarboxylic acid cycle.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8233
- synonyms:
